packages/webview_flutter/webview_flutter/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## NEXT
+## 3.0.2
 
+* Migrate deprecated `Scaffold.showSnackBar` to `ScaffoldMessenger`.
```

Comment:
nit: `Migrate` should be `Migrates` to follow https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style. Also include that the change is only in the example app. This should also be done for `webview_flutter_android` and `webview_flutter_wkwebview`.

```suggestion
* Migrates deprecated `Scaffold.showSnackBar` to `ScaffoldMessenger` in example app.
```